Library/ViewModels/RewardsCollectionViewModel.swift

```diff
@@ -159,3 +186,16 @@ public final class RewardsCollectionViewModel: RewardsCollectionViewModelType,
   public var inputs: RewardsCollectionViewModelInputs { return self }
   public var outputs: RewardsCollectionViewModelOutputs { return self }
 }
+
+private func titleForContext(_ context: RewardsCollectionViewContext) -> String {
+  return context == .createPledge ? Strings.Back_this_project() : Strings.Choose_another_reward()
+}
+
+private func backedReward(_ project: Project, rewards: [Reward]) -> IndexPath? {
```

Comment:
Please update this to use the function @ifbarrera is using in #905 when that merges.

https://github.com/kickstarter/ios-oss/pull/905/files#diff-234cfa58d101616352832cb332756878R33